tests/cpp/testobject/NestedPdxObject.cpp

```diff
@@ -68,6 +68,31 @@ bool ChildPdx::equals(ChildPdx& other) const {
   return false;
 }
 
+ParentPdx::ParentPdx()
+    : m_parentId{-1},
+      m_parentName{},
+      m_childPdx{},
+      m_enum{},
+      m_char{L'\0'},
+      m_charArray{} {}
```

Comment:
Probably another opportunity to rename member variables to conform to style guide.